install-troubleshooting.md

```diff
@@ -0,0 +1,4 @@
+# Installation troubleshooting
+
+Troubleshooting tips for installing an Amazon on Red Hat OpenShift cluster using `moactl`.
+
```

Comment:
We could describe `moactl logs` here

Reply:
Right; the extent of troubleshooting now is 

moactl verify permissions
moactl verify quota
moactl logs